packages/user/src/views/Login.tsx

```diff
@@ -25,7 +25,14 @@ const Login = () => {
 
   const handleSubmit = async (credentials: LoginCredentials) => {
     setLoading(true);
-    const result = await login(credentials);
+
+    const result = await login(credentials).catch((err) => {
+      let errorMessage = t("errors.otherErrors", { ns: "errors" });
```

Comment:
better to do translation at the last i.e when after the message is been decided.